src/modules/keyboardmanager/KeyboardManagerEditorLibrary/ShortcutControl.cpp

```diff
@@ -113,6 +116,20 @@ void ShortcutControl::AddNewShortcutControlRow(StackPanel& parent, std::vector<s
     targetAppTextBox.Width(EditorConstants::ShortcutTableDropDownWidth);
     targetAppTextBox.PlaceholderText(KeyboardManagerEditorStrings::DefaultAppName);
     targetAppTextBox.Text(targetAppName);
+    targetAppTextBox.TextChanged([targetAppTextBox](const auto& sender, auto const& e) {
+        // Alphanumeric characters only
+        const std::wstring& text = std::wstring{ targetAppTextBox.Text() };
+        std::wstring newText;
+        for (auto c : text)
+        {
+            if (isalnum(c) || c == ' ' || c == '.')
```

Comment:
What sort of applications do we support in this field or how do we detect it?
Is it by executable name? If that's the case, executable names can have more kinds of characters, which should be allowed here.

Reply:
> What sort of applications do we support in this field or how do we detect it?
It is compared to executable path or module name (with converting to lower case):
https://github.com/microsoft/PowerToys/blob/master/src/common/utils/process_path.h#L10-L26

I'll update logic according to this
https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#file-and-directory-names

Reply:
Hm, after some investigation, I'm not sure whether this issue is valid at all.. e.g. 'kao@[][]' is completely valid file/app name

Reply:
I'm thinking of closing both this PR and the issue itself.. @jaimecbernardo WDYT?

Reply:
Yes, even if it may seem like the user is adding gibberish, it'd still be a valid application name. And I don't think there's a good way for us to validate this.
Perhaps explain it on the issue to see if we understood it well, while pinging the issue submitter and Deondre to get their feedback if the issue should be closed.
Turn this PR into a draft in the meanwhile, I guess.